docs/application/web/guides/machine-learning/trainer.md

```diff
@@ -0,0 +1,294 @@
+});
+```
+
+You can print the model's structure using `summarize` method:
+
+```javascript
+var summary = model.summarize();
+console.log(summary);
+```
+
+Output:
+
+```
+================================================================================
+          Layer name          Layer type     Input dimension         Input layer
+================================================================================
+        layer_input0               input           1:1:28:28
+--------------------------------------------------------------------------------
+       layer_conv2d1              conv2d           1:1:28:28        layer_input0
+--------------------------------------------------------------------------------
+ layer_conv2d1/activ          activation          1:32:26:26       layer_conv2d1
+--------------------------------------------------------------------------------
+    layer_pooling2d2           pooling2d          1:32:26:26 layer_conv2d1/activ
+--------------------------------------------------------------------------------
+      layer_flatten3             flatten          1:32:25:25    layer_pooling2d2
+--------------------------------------------------------------------------------
+           layer_fc4     fully_connected         1:1:1:20000      layer_flatten3
+--------------------------------------------------------------------------------
+ layer_fc4/activatio          activation           1:1:1:100           layer_fc4
+--------------------------------------------------------------------------------
+           layer_fc5     fully_connected           1:1:1:100 layer_fc4/activatio
+--------------------------------------------------------------------------------
+      cross_softmax0       cross_softmax            1:1:1:10           layer_fc5
+================================================================================
+```
+
+### Create model from a configuration file
+
+Configuration file structure is described in [the guide for Tizen Native API](../../../native/guides/machine-learning/machine-learning-training#create-a-model-from-ini-formatted-file). This is a simplified configuration for the model created in the previous section:
+
+```ini
+[model]
+type = NeuralNetwork
+batch_size = 32
+epochs = 2
+
+[optimizer]
+type = sgd
+learning_rate = 0.01
+
+[layer_input0]
+type = input
+input_shape = 1:28:28
+
+[layer_conv2d1]
+type = conv2d
+filters = 32
+kernel_size = "3, 3"
+activation = relu
+
+[layer_pooling2d2]
+type = pooling2d
+pooling = max
+pool_size = "2, 2"
+
+[layer_flatten3]
+type = flatten
+
+[layer_fc4]
+type = fully_connected
+unit = 100
+activation = relu
+
+[layer_fc5]
+type = fully_connected
+unit = 10
+activation = softmax
+```
+
+To load the configuration use `createModel` function:
+
+```javascript
+var configFile = "documents/model.ini";
+var model = tizen.ml.trainer.createModel(configFile);
+```
+
+Compile the model:
+
+```javascript
+model.compile({
+    loss: "cross",
+});
+```
+
+## Train
```

Comment:
Please revert this to "Training".
I think the word "Training" should stay as it is. If we change it to "Train" it might be a little confusing for the readers.
```suggestion
## Training
```